js/background.js

```diff
@@ -730,6 +730,57 @@
       });
     };
 
+    window.createMediumSizeGroup = async (groupName, members) => {
+      // Create Group Identity
+      const identityKeys = await libsignal.KeyHelper.generateIdentityKeyPair();
+      const groupId = StringView.arrayBufferToHex(identityKeys.pubKey);
```

Comment:
will we need to worry about unique constraints or conflicts in the conversations table?

Reply:
This is ultimately stored on SS, so if there's a conflict, it'll conflict there too